src/coreclr/runtime/arm/WriteBarriers.S

```diff
@@ -179,6 +179,10 @@ GLOBAL_LABEL RhpCheckedAssignRefAVLocation
 LEAF_END RhpCheckedAssignRef\EXPORT_REG_NAME, _TEXT
 .endm
 
+LEAF_ENTRY RhpWriteBarriers, _TEXT
```

Comment:
FWIW this would not work anyway... we compile all code on Apple platforms as `.subsections_via_symbols` and the Apple linker is free to reorder the functions (or discard unused ones if told to). Compiling without `subsections_via_symbols` is not an option due to various bugs in different linker versions. The only other option to keep the desired order is to mark the successive symbols as `.alt_entry`, which also happens to run into linker bugs if not done extremely carefully...